src/components/dropdown-item/daikin-dropdown-item.ts

```diff
@@ -89,38 +102,91 @@ export class DaikinDropdownItem extends LitElement {
   @property({ type: Boolean, reflect: true })
   selected = false;
 
-  @query("button")
-  private _button!: HTMLButtonElement | null;
+  @property({ type: Boolean, reflect: true })
+  selectable = false;
+
+  @query("button,span[role=option]")
+  private _focusableElement!: HTMLElement | null;
 
-  private _handleClick(): void {
+  private _handleClick(event: PointerEvent): void {
     if (this.disabled) {
       return;
     }
 
-    this.dispatchEvent(
-      new Event("select", {
-        bubbles: true,
-      })
-    );
+    this.dispatchEvent(new Event("select", event));
+  }
+
+  private _handleChange(event: Event): void {
+    event.preventDefault();
+    this._emitSelect();
+  }
+
+  private _handleContainerClick() {
+    if (this.disabled) {
+      return;
+    }
+
+    this._emitSelect();
+  }
+
+  private _handleContainerKeydown(event: KeyboardEvent) {
+    if (!isSimilarToClick(event.key) || this.disabled) {
+      return;
+    }
+
+    this._emitSelect();
+  }
+
+  private _emitSelect(): void {
+    this.dispatchEvent(new Event("select", { bubbles: true }));
   }
 
   override render() {
-    return html`<button
-      type="button"
-      class=${cvaOption({ selected: this.selected })}
-      data-value=${this.value}
-      role="option"
-      aria-selected=${this.selected}
-      tabindex="-1"
-      ?disabled=${this.disabled}
-      @click=${this._handleClick}
-    >
-      <slot></slot>
-    </button>`;
+    const optionClassName = cvaOption({
+      selected: this.selected,
+      disabled: this.disabled,
+      multiple: this.selectable,
+    });
+
+    return this.selectable
+      ? html`<span
+          class=${optionClassName}
+          data-value=${this.value}
+          role="option"
+          aria-selected=${this.selected}
+          tabindex="-1"
+          aria-disabled=${this.disabled}
+          @click=${this._handleContainerClick}
```

Comment:
nit. It may be a good idea to prevent text from being selected by double-clicking.

```suggestion
          @click=${this._handleContainerClick}
          @mousedown=${this._handleContainerMousedown}
```

where

```ts
  private _handleContainerMousedown(event: KeyboardEvent) {
    if (event.detail === 2) {
      // Prevent text selection on double click.
      event.preventDefault();
    }
  }
```